lib/slack-ruby-bot-server/models/common_methods.rb

```diff
@@ -0,0 +1,56 @@
+module CommonMethods
```

Comment:
I don't love this module name and location. If this is a team's common functionality it should be something like `Team::Methods`.